index.js

```diff
@@ -63,12 +64,12 @@ class KeepAChangelog extends Plugin {
   }
 
   beforeRelease() {
-    const { keepUnreleased } = this;
+    const { addUnreleased, keepUnreleased } = this;
     const { isDryRun } = this.config;
     if (isDryRun || keepUnreleased) return;
     const { version } = this.getContext();
     const formattedDate = getFormattedDate();
-    const releaseTitle = `${this.EOL}${this.EOL}## [${version}] - ${formattedDate}${this.EOL}`;
+    const releaseTitle = `${addUnreleased ? this.unreleasedTitle : this.EOL}${this.EOL}## [${version}] - ${formattedDate}${this.EOL}`;
```

Comment:
@eMarek  thank you for the feedback. You alright about "unreleased" line in changelog file of actual release, but that is optional. I would to love achievable your request, @webpro can help us?

Reply:
If you mean to add this "Unreleased" section _after_ the changelog file has been committed and pushed, then [this.afterRelease()](https://github.com/release-it/release-it/blob/master/docs/plugins.md#afterrelease) can be used for this.

The (internal) Git plugin does `git commit` and `git push` during the `release` release-cycle method.

Reply:
Yes, this is exactly what I meant.

@elsangedy, yes, I know it is optional. I just wanted to suggest that it would be nice to have both options. One for `beforeRelease` and one for `afterRelease`. This is just my idea. Otherwise I am very pleased with your changes. As far as I am concerned, this can be merged. :)